api/src/main/java/ca/bc/gov/educ/api/distribution/process/MergeProcess.java

```diff
@@ -49,12 +49,12 @@ public ProcessorData fire(ProcessorData processorData) {
 		Long batchId = processorData.getBatchId();
 		int numberOfPdfs = 0;
 		int counter = 0;
-		List<School> schoolsForLabels = new ArrayList<>();
+		List<ca.bc.gov.educ.api.distribution.model.dto.School> schoolsForLabels = new ArrayList<>();
```

Comment:
should be v2?

Reply:
This is right as School DTO here is for Report Data being used by Grad-Report-Api.  Some APIs have it's name as SchoolReport.java.